src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/riscv64/eltwise.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/eltwise.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+#include "utils/filter_cpu_info.hpp"
+#include "nodes/kernels/riscv64/cpu_isa_traits.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace Eltwise {
+namespace {
+
+/*
+ * The motivation of this test is to validate different input and output precisions of Eltwise Op.
+ * If IO data type is not supported by jit emitter, they should be converted
+ * to supported types on operation inputs and outputs in JIT kernel
+*/
+
+static const std::vector<ov::test::utils::EltwiseTypes> ops() {
+    // JIT us supported only when `gcv` is available
+    if (ov::intel_cpu::riscv64::mayiuse(ov::intel_cpu::riscv64::gcv)) {
+        return { utils::EltwiseTypes::ADD };
+    }
+    return {};
+}
+
+const std::vector<ov::AnyMap>& config_infer_prc_f32() {
+    static const std::vector<ov::AnyMap> additionalConfig = {
+        {{ov::hint::inference_precision.name(), ov::element::f32}},
+    };
+    return additionalConfig;
+}
+
+const auto params_4D_jit = ::testing::Combine(
+        ::testing::Combine(
+                ::testing::ValuesIn(static_shapes_to_test_representation(inShapes_4D())),
```

Comment:
Does these shapes cover tails and different lmuls (like for the future than not only m1 is supported)?